lattices/src/set_union_with_tombstones.rs

```diff
@@ -269,6 +273,15 @@ pub type SetUnionWithTombstonesOptionSet<Item> =
 pub type SetUnionWithTombstonesTombstoneOnlySet<Item> =
     SetUnionWithTombstones<EmptySet<Item>, SingletonSet<Item>>;
 
+/// [`crate::tombstone::RoaringTombstoneSet`]-backed tombstone set with [`std::collections::HashSet`] for the main set.
+/// Provides space-efficient tombstone storage for u64 integer keys.
+pub type SetUnionWithTombstonesRoaring = SetUnionWithTombstones<HashSet<u64>, RoaringTombstoneSet>;
```

Comment:
Feels a bit unnecessary to have this alias? The length of the name is almost as long as what it aliases 😄